src/document/DocumentAnnotator.ts

```diff
@@ -58,6 +59,9 @@ export default class DocumentAnnotator extends BaseAnnotator {
     }
 
     render(): void {
+        if (!getVisibility(this.store.getState())) {
```

Comment:
Does this work as events are fired? Does it clean up existing managers properly? Is something calling `render` or `init`?